java/arcs/core/data/expression/Evaluator.kt

```diff
@@ -23,7 +23,7 @@ import arcs.core.util.Time
  * reflected as exceptions.
  */
 class ExpressionEvaluator(
-    val currentScope: Scope = CurrentScope<Any?>(),
+    var currentScope: Scope = CurrentScope<Any?>(),
```

Comment:
Can you add a TODO, we can reuse b/165643903 here

Reply:
Fixed